### PR TITLE
refactor: change redis creation to use url

### PR DESCRIPTION
**Pull request by franciscog-kainos**
### JIRA link
N/A

### Change description
Changed use of redis port and host to use the full url.

### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes look good on mobile
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are commited to keeping commit history clean, consistent and linear. To achive this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: src/ContainerFactory.ts [hunk 1]
```diff
@@ -3,12 +3,8 @@ import { RedisClient, createClient } from 'redis';
 import { buildProviderModule } from 'inversify-binding-decorators';
 
 const createRedisClient = () => {
-    return createClient(
-        {
-            host: process.env.REDIS_HOST,
-            port: Number(process.env.REDIS_HOST)
-        }
-    ).on('error', (err) => { throw err; });
+    const redisUrl = 'redis://' + process.env.CACHE_SERVER || 'localhost:6379';
```
Comment (geek-soft) [inline src/ContainerFactory.ts]: No default please. We expect to fail is config is missing.

Comment (geek-soft) [reply src/ContainerFactory.ts]: Not a big deal but why not to use string interpolation instead of '+'?

Comment (franciscog-kainos) [reply src/ContainerFactory.ts]: Good point, fixed those

File: src/ContainerFactory.ts [hunk 2]
```diff
@@ -3,12 +3,8 @@ import { RedisClient, createClient } from 'redis';
 import { buildProviderModule } from 'inversify-binding-decorators';
 
 const createRedisClient = () => {
-    return createClient(
-        {
-            host: process.env.REDIS_HOST,
-            port: Number(process.env.REDIS_HOST)
-        }
-    ).on('error', (err) => { throw err; });
+    const redisUrl = `redis://${process.env.CACHE_SERVER}`;
```
Comment (geek-soft) [inline src/ContainerFactory.ts]: Ideally if you checked whether `process.env.CACHE_SERVER` is set and throw readable error when not set.

**Pull request merged**